metricbeat/module/ceph/cluster_health/data.go

```diff
@@ -25,17 +25,20 @@ import (
 	"github.com/elastic/beats/libbeat/common"
 )
 
+//Timecheck contains part of the response from a HealthRequest
```

Comment:
Spaces before comments please :slightly_smiling_face: 
```suggestion
// Timecheck contains part of the response from a HealthRequest
```